compose-builder/add-secure-device-mqtt.yml

```diff
@@ -0,0 +1,44 @@
+# /*******************************************************************************
+#  * Copyright 2022 Intel Corporation.
+#  *
+#  * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+#  * in compliance with the License. You may obtain a copy of the License at
+#  *
+#  * http://www.apache.org/licenses/LICENSE-2.0
+#  *
+#  * Unless required by applicable law or agreed to in writing, software distributed under the License
+#  * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  * or implied. See the License for the specific language governing permissions and limitations under
+#  * the License.
+#  *
+#  *******************************************************************************/
+
+version: '3.7'
+
+services:
+  device-mqtt:
+    image: ${DEVICE_SVC_REPOSITORY}/device-mqtt${ARCH}:${DEVICE_MQTT_VERSION}
+    ports:
+      - "127.0.0.1:59982:59982"
+    container_name: edgex-device-mqtt
+    hostname: edgex-device-mqtt
+    read_only: true
+    restart: always
+    networks:
+      - edgex-network
+    env_file:
+      - common.env
+      - device-common.env
+    environment:
+      SERVICE_HOST: edgex-device-mqtt
+      MQTTBROKERINFO_HOST: edgex-mqtt-broker
+      MQTTBROKERINFO_AUTHMODE: usernamepassword
+      MQTTBROKERINFO_CREDENTIALSPATH: message-bus
```

Comment:
This is all that is needed in the extension. The rest is duplicate of what is in add-device-mqtt.yml.

Reply:
yes, now it's an extension